source/math/IntRandomGenerator.ooc

```diff
@@ -15,17 +15,35 @@
 // along with this program. If not, see <http://www.gnu.org/licenses/>.
 
 use ooc-math
+import math
 import os/Time
 
-IntRandomGenerator: class {
-	_backend: FloatRandomGenerator
-	init: func ~withBackend (=_backend)
-	free: override func {
-		this _backend free()
-		super()
+IntUniformRandomGenerator: class {
+	_state, _min, _max, _range: Int
+	init: func (seed := Time microtime()) {
+		this _min = 0
+		this _max = 32767
```

Comment:
Having a fast way to return values in the `Short` range is nice, but I don't think it should be the **default** behavior. If you create an `IntRandomGenerator`, you should be able to assume it will return values in the `Int` range unless otherwise specified, which is what the other constructor is for.